acceptance/tests/facts/windows.rb

```diff
@@ -22,7 +22,10 @@
     os_version  = '2012 R2'
     kernel_version = '6.3'
   elsif agent['platform'] =~ /-10/
-    os_version  = /^10\./
+    os_version  = '10'
```

Comment:
If we were intentionally returning the full kernel string on consumer Windows 10, then we need to change the resolver code to continue to do that. But I doubt that was the case.